### DISCUSSION TITLE
Problems from the "Big Table" and inconsistency between web and app

### DISCUSSION THREAD

**Discussion by serbasan** [doudar/SmartSpin2k#579] [Q&A]
Hi Anthony, I have some questions and doubts:

1) I have the SmartSpin2k app installed on my phone and I can modify the Incline Multiplier to values ​​less than 1 and on the web it only goes up to 1.
With Ship Step it's the opposite, in the app I can go down to 100 and on the web up to 50.

This creates inconsistency if you record from the web or from the app. Can it be fixed? It would be nice to allow values ​​less than 1 for Incline Multiplier and less than 50 for Ship Step both in the app and on the web.

2) I was testing the new firmware that includes the new "Big Table" and I had to remove it because the motor moved to the end of both extremes especially in sudden power changes. The Incline Multiplier was at 1 and in Zwift it was at half. (all by default).

With 24.7.9, which does not use the Big Table, it works perfectly for me and even better if I lower the Incline Multiplier from Zwift (I no longer have to shift down and up so much and it is much more pleasant)

I will test with the latest firmware with that parameter lower in Zwift to see if it does not reach the limits. I remember that my configuration is very particular since the Keiser M3 is very sensitive, with small changes to the Motor the resistance changes a lot.

3) Another question, how does the Big Table improve the previous configuration? When the table is built, is it permanent between restarts or must it be redone on each ride?

Greetings.

**Comment by doudar** [comment]
Big Table is potentially a little more aggressive in seeking to find ERG setpoints. I just merged a change into develop that might help that somewhat, so feel free to try it out. As far as aligning the parameters, feel free to take a look at #581 - I made those same changes in the ConfigApp (already merged there). The app will take a few days to publish.
